chcemvediet/apps/accounts/admin.py

```diff
@@ -2,14 +2,28 @@
 # -*- coding: utf-8 -*-
 from django import forms
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+from django.contrib.auth.models import User
 from django.core.exceptions import ValidationError
 
-from poleno.utils.misc import decorate
+from poleno.utils.admin_login_as import AdminLoginAsAdminMixin
 from poleno.utils.admin import simple_list_filter_factory, admin_obj_format
+from poleno.utils.misc import decorate
 
 from .models import Profile
 
 
+admin.site.unregister(User)
+@admin.register(User, site=admin.site)
+class UserAdmin(AdminLoginAsAdminMixin, DjangoUserAdmin):
+    list_display = DjangoUserAdmin.list_display + (
+            decorate(
+                lambda o: admin_obj_format(o, u'Log in', link=u'login_as'),
+                short_description=u'Login As',
+                ),
+            )
+    LOGIN_AS_REDIRECT_VIEWNAME = u'inforequests:mine'
```

Comment:
@viliambalaz By som ten atribut nazval normalne lower case. Nie je to globalna konstanta. Aj ostatne atributy v admin classe su lower case, ci nie?

Reply:
:ok: